var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -71,18 +100,38 @@ class Kokkos(Package):
         description='Set the GPU architecture to use'
     )
 
+    # Checks on Kokkos version and Kokkos options
+    conflicts('+aggressive_vectorization', when='@0:2.0.99',)
```

Comment:
You can drop the `0`, open range, i.e. `@:2.0.99` should work as well!

Reply:
Cheers! Added per suggestion in forthcoming commit.